docs/developers-guide.md

```diff
@@ -24,24 +24,27 @@ If you have problems with your development environment, make sure that you are n
 
 These are the set of tools which are required in order to complete any build of the Metabase code. Follow the links to download and install them on your own before continuing.
 
-1. [Java Development Kit JDK (https://adoptopenjdk.net/releases.html)](https://adoptopenjdk.net/releases.html) - latest LTS version of JDK - more about [Java versions](./operations-guide/java-versions.md)
-2. [Node.js (http://nodejs.org/)](http://nodejs.org/) - latest LTS release
-3. [Yarn package manager for Node.js](https://yarnpkg.com/) - latest release of version 1.x
-4. [Leiningen (http://leiningen.org/)](http://leiningen.org/) - latest release
+1. [Clojure (ttps://clojure.org)](https://clojure.org/guides/getting_started) - install the latest release by following the guide depending on your OS
+2. [Java Development Kit JDK (https://adoptopenjdk.net/releases.html)](https://adoptopenjdk.net/releases.html) - you need to install JDK 8 (./operations-guide/java-versions.md)
+3. [Node.js (http://nodejs.org/)](http://nodejs.org/) - latest LTS release
+4. [Yarn package manager for Node.js](https://yarnpkg.com/) - latest release of version 1.x - you can install it in any OS by doing `npm install --global yarn`
+5. [Leiningen (http://leiningen.org/)](http://leiningen.org/) - latest release
+6. [GetText package (https://www.gnu.org/software/gettext/)](https://www.gnu.org/software/gettext/) - latest
 
-On a most recent stable Ubuntu/Debian, the above tools can be installed by using:
+On a most recent stable Ubuntu/Debian, all the tools above, with the exception of Clojure and JRE 11, can be installed by using:
```

Comment:
It says "install JDK 8" above -- probably shouldn't say "JRE 11" here

Reply:
you're absolutely right